packages/lesmis/src/api/queries/useVitalsData.js

```diff
@@ -85,23 +85,19 @@ const useEntityReport = entity =>
   );
 
 const useSchoolVitals = entity => {
-  const { data, isLoading } = useSchoolReport(entity);
+  const { data: results, isLoading } = useSchoolReport(entity);
 
   return {
-    data: {
-      ...data?.results[0],
-    },
+    data: results ? results[0] : undefined,
```

Comment:
I think you could use 
```suggestion
    data: results?[0],
```
but also clear and good as is

Reply:
I tried this and it didn't work so I swapped it out quickly, just took another look an realised the format is just slightly unexpected for arrays `results?.[0]` seems to work

Reply:
Ah right! That is just slightly unexpected.